lib/scss/layouts/stanford-page.layout.scss

```diff
@@ -0,0 +1,8 @@
+@charset 'UTF-8';
+@import '../jumpstart_ui.config';
+
+.node-stanford-page-su-page-components {
+  @media #{$su-breakpoint-xl} {
```

Comment:
This should use the `@import grid-media('xl') {` mixin format.

Reply:
Got it!